app/ui/browser/views/browser.jsx

```diff
@@ -82,8 +82,9 @@ class BrowserWindow extends Component {
       }));
     };
     const onLocationContextMenu = e => menuLocationContext(e.target, currentPageId, dispatch);
+    const onClearCompletions = () => dispatch(actions.clearCompletions());
     const onLocationReset = () => {
-      dispatch(actions.setUserTypedLocation(currentPageId, { text: void 0 }));
+      dispatch(actions.locationChanged(currentPageId, { text: null }));
```

Comment:
I don't think there was a reason - I believe locationChanged was added after the fact and this never got changed b/c there was no difference between the two actions at the time